evals/util.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import base64
+import mimetypes
+import os
+from typing import List
+
+import pandas as pd
+from llama_stack_client.lib.agents.agent import Agent
+from tqdm import tqdm
+
+
+async def get_response_row(agent: Agent, input_query: str) -> str:
```

Comment:
not for this PR maybe but can we push this into the client-lib? @subramen has pointed a need for this and other related helpers.

Reply:
Yeah, this seems like a useful util function for response parsing.